NuRadioReco/modules/trigger/highLowThreshold.py

```diff
@@ -215,12 +216,14 @@ def run(self, evt, station, det,
             Options:
             * 'voltage' to store the ADC output as discretised voltage trace
             * 'counts' to store the ADC output in ADC counts
+        pre_trigger_time: float (default: 55 ns)
+            The time data before the trigger occured is stored (i.e., defines the readout window).
```

Comment:
I don't mind whether this or #940 gets merged first, but in either case please document also the option to use a `dict` to set the pre trigger times, and/or link to the documentation of this feature in `framework.trigger.Trigger`.